File: PRAK405-2310817210001-AndraBraputraAkbarSaleh.py
a, b = map(int, input("").split())
total = 0
total2 = 0

for i in range(1, a + 1):
    total += i * b
    output = "(" + ") + (".join([f"{j} * {b}" for j in range(i, 0, -1)]) + ")"
    print(f"{output} = {total}")
    total2 += total

print(total2)

File: PRAK405-2310817210001-AndraBraputraAkbarSaleh.c
#include <stdio.h>

int main() {
    int i, j, a, b, c, total = 0, total2 = 0;

    scanf("%d %d", &a, &b);

        for (i = 1; i <= a; i++) {
        c = i * b;
        printf("(");
        for (j = i; j >= 1; j--) {
            printf("%d * %d", j, b);
            if (j != 1) {
                printf(") + (");

            }
        }
        total += c;
        printf(") = %d\n", total);
        total2 +=total;
        }
        printf("%d\n", total2);   
    return 0;
}